src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/AbsoluteUriInRelatedLocation.sarif

```diff
@@ -10,25 +10,13 @@
         {
           "ruleId": "TST0001",
           "level": "error",
-          "formattedRuleMessage": {
-            "formatId": "Default"
-          },
-          "locations": [
```

Comment:
>locations [](start = 11, length = 9)

why did this go?

Reply:
Each test file has only and exactly the elements it needs to demonstrate the proposition stated by the test name. I don't need `result.locations` to demonstrate that I can detect an unwanted absolute URI in `relatedLocations`. I'm trying to keep the test content as small and focused as possible.

---
In reply to: [210452611](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210452611) [](ancestors = 210452611)

Reply:
Right but this approach leads to file/test proliferation. Alternate philosophy drives maximal efficiency from least content. I prefer this for data-driven approaches. Use the property bag to store important clarifying information for individual results. Just my opinion. If you want small, focused testing, your string literals approach makes more sense. 

obviously your mileage varies philosophically. 

---
In reply to: [210452997](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210452997) [](ancestors = 210452997,210452611)